dask/array/backends.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
 
+from dask.array.percentile import _percentile
```

Comment:
Nit: could you update this to `from .percentile import _percentile` to match all the other internal imports in this module?